cosmos-gui/src/app_utils.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-tidoop (FI-WARE project).
+ *
+ * fiware-tidoop is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-tidoop is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-tidoop. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with
+ * francisco dot romerobueno at telefonica dot com
+ */
+
+/**
+ * Main app utils.
+ *
+ * Author: frb
+ */
+
+// Module dependencies
+var boom = require('express-boom');
+var cmdRunner = require('./cmd_runner.js');
+var logger = require('./logger.js');
+
+function provisionCluster(res, clusterPrivKey, clusterUser, clusterEndpoint, hdfsSuperuser, hdfsQuota, username, password) {
+    cmdRunner.run('bash', ['-c', 'echo "sudo useradd ' + username + '" | ssh -i ' + clusterPrivKey + ' ' + clusterUser
+    + '@' + clusterEndpoint], function(error, result) {
+        if (error) {
+            res.boom.badData('There was an error while adding the Unix user ' + username, error);
```

Comment:
After checking the express-boom module, let's try to avoid using this module since it does not add much value :)

Instead of the previous code, try the following:
  1. Instead of `var boom = require('express-boom');`, let's require boom directly: `var boom = require('boom');` (remember tu `npm uninstall express-boom` and `npm install boom --save`
  2. Instead of `res.boom.badData('There was an error while adding the Unix user ' + username, error);`, let's do the following:
```javascript
var boomError = boom.badData('There was an error while adding the Unix user ' + username, error);
res.status(boomError.output.statusCode).send(boomError.output.payload.message);
```
Tell me how it goes... :)